requirements/prod.txt

```diff
@@ -85,6 +84,4 @@ wcwidth==0.2.6
 WebOb==1.8.8
 Werkzeug==3.0.3
 zipp==3.20.2
-zope.event==4.5.0
-zope.interface==5.4.0
```

Comment:
Dependencies of [gevent](https://github.com/gevent/gevent/blob/master/setup.py#L231-L238) .